apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { PopoverProps } from '@fluentui/react-popover';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('#info-button')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .focus('#info-button')
+        .snapshot('focus', { cropTo: '.testWrapper' })
```

Comment:
It should have a border when focused, right? It's not showing a border in the screener results:
![image](https://user-images.githubusercontent.com/48106640/202536419-da8a4e1a-237a-470b-b58f-ff26a23d9153.png)